src/Evolution/Systems/ScalarWave/Equations.hpp

```diff
@@ -241,23 +257,25 @@ struct UpwindFlux {
   // `()` operator.
   using package_tags =
       tmpl::list<::Tags::NormalDotFlux<Pi>, ::Tags::NormalDotFlux<Phi<Dim>>, Pi,
-                 NormalTimesFluxPi>;
+                 NormalTimesFluxPi, Gamma2Psi, NormalTimesGamma2Psi>;
 
   // These tags on the interface of the element are passed to
   // `package_data` to provide the data needed to compute the numerical fluxes.
   using argument_tags =
       tmpl::list<::Tags::NormalDotFlux<Pi>, ::Tags::NormalDotFlux<Phi<Dim>>, Pi,
+                 Psi, Tags::ConstraintGamma2,
                  ::Tags::Normalized<::Tags::UnnormalizedFaceNormal<Dim>>>;
 
   // pseudo-interface: used internally by Algorithm infrastructure, not
   // user-level code
   // Following the not-null pointer to packaged_data, this function expects as
   // arguments the databox types of the `argument_tags`.
   void package_data(
-      gsl::not_null<Variables<package_tags>*> packaged_data,
+      const gsl::not_null<Variables<package_tags>*> packaged_data,
```

Comment:
Because this is a declaration, I don't think you need to add the `const` here

Reply:
I was about to push a fix but you got there first :)